build.gradle

```diff
@@ -39,10 +39,6 @@ dependencies {
   implementation "org.springframework.boot:spring-boot-starter-amqp"
 
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-elasticsearch', version: '2.4.5'
```

Comment:
No need to provide a version, let the spring boot parent decide.

Reply:
Version is removed. Used double quote instead of single quote to keep consistency.